sdk/storage/Azure.Storage.Files/src/DirectoryClient.cs

```diff
@@ -1328,21 +1397,20 @@ private void SetNameFieldsIfNull()
         /// A <see cref="StorageRequestFailedException"/> will be thrown if
         /// a failure occurs.
         /// </remarks>
-        public virtual async Task<Response<StorageClosedHandlesSegment>> ForceCloseHandlesAsync(
-            string handleId = Constants.CloseAllHandles,
+        public virtual async Task<Response<StorageClosedHandlesSegment>> ForceCloseAllHandlesAsync(
             string marker = default,
             bool? recursive = default,
             CancellationToken cancellationToken = default) =>
```

Comment:
Nit - can we swap the parameter ordering for `marker` and `recursive`?  We should leave marker just in case for completeness, but it's pretty useless now that we're looping for the user.